modules/back-end/src/Domain/SemanticPatch/Instruction.Flag.Variation.cs

```diff
@@ -76,17 +76,17 @@ public override void Apply(FeatureFlag flag)
     }
 }
 
-public class DefaultVariationInstruction : FlagInstruction
+public class UpdateDefaultRuleVariationOrRolloutInstruction : FlagInstruction
 {
-    public DefaultVariationInstruction(Fallthrough value) : base(FlagInstructionKind.UpdateDefaultVariation, value)
+    public UpdateDefaultRuleVariationOrRolloutInstruction(DefaultRuleRolloutVariations value) : base(FlagInstructionKind.UpdateDefaultRuleVariationOrRollouts, value)
```

Comment:
I would prefer use `DefaultVariationsInstruction`

Reply:
DefaultVariation is not splitted to **DefaultRuleVariationOrRollout** and **DefaultRuleDispatchKey**